005_prime.js

```diff
@@ -1,10 +1,10 @@
 const prime = (n) => {
   for (let x = 2; x < n; x++) {
-    if (n % x == 0) {
+    if (n % x === 0) {

```

Comment:
Logic is wrong. Did you check all possible outcomes. If the input is 9, this code says its a prime number. Actually 9 is not a prime number. So cross check the function 's logic.